src/web/index.md

```diff
@@ -88,7 +88,7 @@ The following resources can help you get started:
 [Flutter Gallery]: {{site.gallery}}
 [main Flutter repo]: {{site.repo.flutter}}
 [Preparing an app for web release]: {{site.url}}/deployment/web
-[Progressive Web Application]: {{site.developers}}/web/progressive-web-apps/
+[Progressive Web Application]: https://web.dev/progressive-web-apps/
```

Comment:
I suspect @sfshaza2 will want a var for `https://web.dev`

Reply:
We don't have one for flutter.github.io 🤷

Reply:
I can live with this.  :). But, yeah, if we used it often, I'd want a macro.

Reply:
Too bad we don't have something more Flutter-specific to point to.